react-todo-app/src/components/TodoItem.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState, useEffect } from "react"
+import styles from "./TodoItem.module.css"
+import { FaTrash } from "react-icons/fa";
+
+const TodoItem = (props) => {
+  const [editing, setEditing] = useState(false)
+
+  useEffect(() => {
+    return () => {
+      console.log("Cleaning up...")
+    }
+  }, [])
```

Comment:
I think it would be better if you do not commit `console.log` to your repo. it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website. so kindly remove this part.
- check [Js best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)